centos6/Dockerfile

```diff
@@ -48,6 +48,8 @@ RUN \
     yum clean metadata && \
     yum install -y \
         apache-maven && \
+    curl -sL https://rpm.nodesource.com/setup_12.x | bash - && \
```

Comment:
CentOS 6 has reached end of life and this is not really needed anymore.

Reply:
ive removed my change from the centos6 Dockerfile